components/Item/ItemMenu.vue

```diff
@@ -48,7 +48,7 @@ export default Vue.extend({
     items: {
       get(): MenuItem[] {
         const menuItems = [] as MenuItem[];
-        if (this.$auth.$state.user.Policy.IsAdministrator) {
+        if (this.$auth?.$state?.user?.Policy?.IsAdministrator) {
```

Comment:
```suggestion
        if (this.$auth?.user?.Policy?.IsAdministrator) {
```
It should work
![print_screen_2021-01-05-16-33-20](https://user-images.githubusercontent.com/1619359/103665483-c4694a00-4f73-11eb-99f8-035b06f60dbf.png)

Reply:
@ThibaultNocchi Good catch